pkgs/development/libraries/pipewire/default.nix

```diff
@@ -241,9 +241,12 @@ stdenv.mkDerivation (finalAttrs: {
   doCheck = true;
   doInstallCheck = true;
 
-  postUnpack = ''
-    patchShebangs ${finalAttrs.src.name}/doc/*.py
-    patchShebangs ${finalAttrs.src.name}/doc/input-filter-h.sh
+  postPatch = ''
+    patchShebangs doc/*.py
+    patchShebangs doc/input-filter-h.sh
+
+    # Remove installed-test that runs forever
```

Comment:
I know this is very nitpicky, but can we not batch this with unrelated changes?

Reply:
Would it be better to split the commit here or create a new PR?